tools/provision/chocolatey/openssl.ps1

```diff
@@ -88,15 +93,20 @@ if (-not (Test-Path $zipFile)) {
   Invoke-WebRequest $url -OutFile "$zipFile"
 }
 
```

Comment:
Add in logic to derive and create the build path:
```
$loc = Get-Location
# Keep the location of build script, to bring with in the chocolatey package
$buildScriptSource = $MyInvocation.MyCommand.Definition

# Create the choco build dir if needed
$buildPath = Get-OsqueryBuildPath
if ($buildPath -eq '') {
  Write-Host '[-] Failed to find source root' -foregroundcolor red
  exit
}
$chocoBuildPath = "$buildPath\chocolatey\$packageName"
if (-not (Test-Path "$chocoBuildPath")) {
  New-Item -Force -ItemType Directory -Path "$chocoBuildPath"
}
Set-Location $chocoBuildPath
```

Reply:
This is all already in there L80-92. Unless Im confused about what you're requesting 😕